default.hbs

```diff
@@ -84,6 +84,27 @@ $(document).ready(function () {
     $('.gh-burger').click(function () {
         $('body').toggleClass('gh-head-open');
     });
+
+    // Filter
+    $('.filter-selection').click(function(event) {
+        let classToFilter = event.target.innerHTML;
+        let internalTag = $(this).hasClass('internal-tag');
+        // Basic slug generator: replace whitespaces with dashes, then strip punctuation
+        // TODO: Document behaviour of this generator in API v3
+        classToFilter = classToFilter.replace(/\s+/g, '-').replace(/[.,'!?]/, '').toLowerCase();
+        console.log(classToFilter);
```

Comment:
Could this `console.log` be removed? Other than that, all looks good. Thanks!

Reply:
My bad didn't catch that, fixed